src/bundle/Resources/public/scss/_content-edit.scss

```diff
@@ -1,10 +1,18 @@
+.ez-content-edit {
+    h2,
+    h4 {
+        font-size: 1.125rem;
+        color: $ez-color-base-light;
+    }
+}
+
 .ez-content-edit-container {
     position: relative;
 
     &__close {
         position: absolute;
-        right: 1rem;
-        top: 1rem;
+        left: 0;
```

Comment:
Have you checked how the X button behaves on a different screen resolutions? Please, try resizing the browser window. As far as I know, the margin on the left side is not always present, so the X button will be placed over the content edit header element.